src/commands/build/watch/mod.rs

```diff
@@ -24,14 +24,17 @@ pub fn watch_and_build(
     let project_type = &project.project_type;
     match project_type {
         ProjectType::JavaScript => {
-            let package = Package::new("./")?;
-            let entry = package.main()?;
+            let path = "./";
+            let package = Package::new(path)?;
+            // Confirm entrypoint is provided
```

Comment:
Why do we need to confirm that an entrypoint is provided? I would assume it will fail during the build, but the watch shouldn't care?

Reply:
OOh, this is a good point. I will remove this logic.